content/get-started/quickstart/contributing-to-projects.md

```diff
@@ -95,6 +95,34 @@ gh repo fork REPOSITORY --clone=true
 
 {% enddesktop %}
 
+## Creating a branch to work on
+
+Before making changes to the project, you should create a new branch to commit them to. By keeping changes in their own branch, you follow [GitHub Flow](/get-started/quickstart/github-flow#following-github-flow) and ensure that it will be easier to contribute to the same project again in the future.
```

Comment:
I think we should probably mention here that you need to move to the new branch, as well as creating it. That way the git commands will make more sense.

I've also suggested an update this to bring the link into line with our [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#links). 

```suggestion
Before making changes to the project, you should create a new branch and check it out. By keeping changes in their own branch, you follow GitHub Flow and ensure that it will be easier to contribute to the same project again in the future. For more information, see "[GitHub Flow](/get-started/quickstart/github-flow#following-github-flow)."
```